### PR TITLE
Forward/rel/6.4

### PR DESCRIPTION
**Pull request by LaurenSpiegel**

### CODE REVIEW

**Comment by LaurenSpiegel** [comment]
@ironman-machine go

File: lib/api/apiUtils/bucket/bucketDeletion.js [hunk 1]
```diff
@@ -129,8 +129,11 @@ export function deleteBucket(bucketMD, bucketName, canonicalID, log, cb) {
         // delete a bucket even if there are ongoing multipart uploads.
         function deleteMPUbucketStep(next) {
             const MPUBucketName = `${constants.mpuBucketPrefix}${bucketName}`;
+            // check to see if there are any mpu overview objects (so ignore
+            // any orphaned part objects)
             return metadata.listObject(MPUBucketName,
-                { maxKeys: 1 }, log, (err, objectsListRes) => {
+                { maxKeys: 1, prefix: 'overview' }, log,
```
Comment (ploki) [inline lib/api/apiUtils/bucket/bucketDeletion.js]: no slash on ```overview```?

File: tests/unit/api/bucketDelete.js [hunk 2]
```diff
@@ -61,6 +76,68 @@ describe('bucketDelete API', () => {
...
+                    uploadId: testUploadId,
+                },
+                calculatedHash,
+            }, postBody);
+            objectPutPart(authInfo, partRequest, undefined, log, err => {
+                assert.strictEqual(err, null);
+                const mpuBucketKeyMap =
+                    metadataMem.metadata.keyMaps.get(mpuBucket);
+                assert.strictEqual(mpuBucketKeyMap.size, 2);
+                const overviewKey = `overview${constants.splitter}` +
```
Comment (ploki) [inline tests/unit/api/bucketDelete.js]: ...because of this

File: lib/api/apiUtils/bucket/bucketDeletion.js (hunk 1 above)
Comment (LaurenSpiegel) [inline lib/api/apiUtils/bucket/bucketDeletion.js]: because '/' is not a good splitter for s3 since any character is permitted in an object key.

Comment (ploki) [inline lib/api/apiUtils/bucket/bucketDeletion.js]: obviously 👍

**Pull request merged**